pkg/auth/handler/webapp/viewmodels/base.go

```diff
@@ -217,6 +219,8 @@ func (m *BaseViewModeler) ViewModel(r *http.Request, rw http.ResponseWriter) Bas
 		},
 		ForgotPasswordEnabled:       *m.ForgotPassword.Enabled,
 		PublicSignupDisabled:        m.Authentication.PublicSignupDisabled,
+		AuthenticationDisabled:      m.AuthUI.AuthenticationDisabled && clientid.GetClientID(r.Context()) != "portal",
```

Comment:
Why do we need `clientid.GetClientID(r.Context()) != "portal"`?

Reply:
Otherwise the portal admin will be locked out as well when the authgear is self-hosted or developed locally?

Reply:
We can change the configuration depending on situation. Do not add development hacks.

Reply:
Removed